### DISCUSSION TITLE
Is `rbenv` restricted to a single `system` Ruby?

### DISCUSSION THREAD

**Discussion by lpar** [rbenv/rbenv#1588] [Q&A]
I've got Ruby installed via Homebrew, plus the old Ruby 2.6 shipped with macOS. When I run `rbenv versions` I only get a single entry for `system` (plus the versions installed using rbenv). Switching to the `system` version gets me the Homebrew-installed version, which is before the Apple Ruby in my path.

So it looks as if `rbenv` stops as soon as it finds the first Ruby in the path, and calls that `system`. Is that correct?

**Comment by mislav** [comment]
Yes. "system" is rbenv vocabulary for whatever Ruby version would get activated if you ran `ruby` without rbenv. It's the first Ruby found in PATH.
